src/MvcTables.Samples/Views/Northwind/Orders.cshtml

```diff
@@ -4,9 +4,11 @@
 }
 <section id="Result" class="spy-on-me">
     <h4>Result</h4>
-    
+    @table.PageSizer()
+    @table.Pagination()
     @table.Table()
     @table.Pagination()
+    @table.PageSizer()
```

Comment:
So that you have both the page size and the pagination at the top and the bottom of the table, it's helpful when viewing a large table